ui-v2/app/services/repository/service.js

```diff
@@ -6,8 +6,8 @@ export default RepositoryService.extend({
     return modelName;
   },
   shouldReconcile: function(method) {
-    switch (method) {
-      case 'findGatewayBySlug':
+    switch (true) {
+      case method === 'findGatewayBySlug' || method.indexOf('for-service') !== -1:
```

Comment:
`method` can be either a camelCase or hyphen-delimited string? This should likely be an `if` statement instead of `switch(true)`...

Reply:
Hey @mikemorris ,

> Seems odd that method could be either a camelCase or hyphen-delimited string?

There were two different ways of starting up a blocking query, part of the work proceeding this was to bring this back to there only being one way to start up a blocking query, no matter which ember 'thing' you were doing it from (routes/controllers/components/templates). We did it like this originally to avoid going off on a tangent whilst trying to move forwards with a feature a while back in https://github.com/hashicorp/consul/pull/7967 .

> The current way of marking these types of responses are soon to be moved to the adapters so this kind of configuration is kept all together with the rest of the request configuration.

I'll be PRing this 'do it in the adapter' approach shortly hopefully. If everything goes well there'll be no need to merge this anymore. This PR was only incase it was required whilst I wasn't around.

If that other PR doesn't happen then we can merge this in.